Sources/SwagGenKit/CodeFormatter.swift

```diff
@@ -419,7 +421,13 @@ public class CodeFormatter {
 
         context["required"] = property.required
         context["optional"] = !property.required || property.schema.metadata.nullable
-        context["name"] = getName(property.name)
+      
+        var name = getName(property.name)
+        if let propertyName = propertyNames[name] {
+          name = propertyName
```

Comment:
I would think you’d  need to specify the original name in the swagger spec.
```swift
context["name"] = propertyNames[property.name] ?? getName(property.name)
```

Reply:
yes, I have modified it